spec/checks/checks_shared_examples.rb

```diff
@@ -0,0 +1,63 @@
+require_relative '../spec_helper'
```

Comment:
I'd like to see the factoring-out of shared examples as a separate commit, personally, so the new functionality isn't muddled in with moving-around of old.